rest/tests/test_rest.py

```diff
@@ -0,0 +1,150 @@
+"""
+Unit tests for REST api
+"""
+
+from __future__ import unicode_literals
+
+import json
+
+from learningresources.tests.base import LoreTestCase
+from taxonomy.api import create_vocabulary
+from taxonomy.models import Vocabulary
+from rest.serializers import VocabularySerializer
+
+HTTP_OK = 200
+HTTP_CREATED = 201
+HTTP_NO_CONTENT = 204
+HTTP_UNAUTHORIZED = 403
+
+
+class TestRest(LoreTestCase):
+    """
+    REST test
+    """
+
+    def test_vocabulary(self):
+        """
+        Test for Vocabulary
+        """
+
+        resp = self.client.get('/api/v1/vocabularies/')
+        self.assertEqual(HTTP_OK, resp.status_code)
+        vocabularies = json.loads(resp.content.decode('utf-8'))
+        self.assertEqual(0, vocabularies['count'])
+
+        input_dict = {
+            'name': 'name',
+            'description': 'description',
+            'required': True,
+            'weight': 1000,
+            'vocabulary_type': 'f',
+            'repository': self.repo.id,
+        }
+        resp = self.client.post('/api/v1/vocabularies/', input_dict)
```

Comment:
There was a discussion about trailing slashes in the taxonomy URLs issue. Basically Django will redirect URLs without the trailing slash to include the trailing slash if only that URL is defined, but not vice versa, so it's the more consistent option.